nexus/reconfigurator/planning/src/mgs_updates/test_helpers.rs

```diff
@@ -129,6 +132,22 @@ pub(super) const ARTIFACT_HASH_HOST_PHASE_1_V1: ArtifactHash =
 pub(super) const ARTIFACT_HASH_HOST_PHASE_2_V1: ArtifactHash =
     ArtifactHash([33; 32]);
 
+/// Hash of fake artifact for fake cosmo-b SP
+pub(super) const ARTIFACT_HASH_SP_COSMO_B: ArtifactHash =
+    ArtifactHash([34; 32]);
+/// Hash of fake artifact for fake cosmo RoT slot A
+pub(super) const ARTIFACT_HASH_ROT_COSMO_A: ArtifactHash =
+    ArtifactHash([35; 32]);
+/// Hash of fake artifact for fake cosmo RoT slot B
+pub(super) const ARTIFACT_HASH_ROT_COSMO_B: ArtifactHash =
+    ArtifactHash([36; 32]);
+/// Hash of fake artifact for fake cosmo RoT bootloader
+pub(super) const ARTIFACT_HASH_ROT_BOOTLOADER_COSMO: ArtifactHash =
+    ArtifactHash([37; 32]);
+/// Hash of fake artifact for host OS phase 1
+pub(super) const ARTIFACT_HASH_COSMO_HOST_PHASE_1: ArtifactHash =
+    ArtifactHash([29; 32]);
```

Comment:
This hash is the same as `ARTIFACT_HASH_GIMLET_HOST_PHASE_1`, is there a reason for them being the same? If not, it would be nice to have them be different to make it easier to catch test failures